File: eRecipes/eRecipes.Service/Database/Notifikacije.cs
using System;
using System.Collections.Generic;

namespace eRecipes.Service.Database;

public partial class Notifikacije
{
    public int NotifikacijeId { get; set; }
    public string Naslov { get; set; } = null!;
    public string Sadrzaj { get; set; } = null!;
    public int KorisnikId { get; set; }
    public DateTime DatumSlanja { get; set; } = DateTime.Now;
    public int? ReceptId { get; set; }
    public virtual Recept? Recept { get; set; }
    public virtual Korisnik Korisnik { get; set; } = null!;
    public bool Procitano { get; set; } = false;
}

File: eRecipes/UI/erecipes_desktop/lib/models/notifikacije.dart
import 'package:erecipes_desktop/models/korisnik.dart';
import 'package:json_annotation/json_annotation.dart';

part 'notifikacije.g.dart';

@JsonSerializable()
class Notifikacije{
  int? notifikacijeId;
  String? naslov; 
  String? sadrzaj;
  int? korisnikId;
  Korisnik? korisnik;
  DateTime? datumSlanja;
  bool? procitano;
  
     Notifikacije({
    this.notifikacijeId,
    this.naslov,
    this.sadrzaj,
    this.korisnikId,
    this.korisnik,
    this.datumSlanja,
    this.procitano,
  });
  
  factory Notifikacije.FromJson(Map<String,dynamic> json)=> _$NotifikacijeFromJson(json);

  Map<String,dynamic> toJson() => _$NotifikacijeToJson(this);
}